Xamarin.Forms.Platform.UAP/CollectionView/SelectableItemsViewRenderer.cs

```diff
@@ -162,13 +162,15 @@ void UpdateFormsSelection()
 
 		void UpdateFormsSingleSelection()
 		{
-			var selectedItem = ListViewBase.SelectedItem is ItemTemplateContext itemPair 
-				? itemPair.Item 
+			var selectedItem = ListViewBase.SelectedItem is ItemTemplateContext itemPair
+				? itemPair.Item
 				: ListViewBase.SelectedItem;
-			
+
 			ItemsView.SelectionChanged -= FormsSelectionChanged;
 			ItemsView.SelectedItem = selectedItem;
-			ItemsView.SelectionChanged += FormsSelectionChanged;
+
+			if (ItemsView != null)
```

Comment:
Why we do not check the ItemsView is not null in Line#169?

Reply:
Yeah. It looks like the entire block should be checked and then maybe fail-fast? No need to even do any work if there is no `ItemsView`